src/Protocol/DFRN.php

```diff
@@ -2431,6 +2432,8 @@ private static function processEntry($header, $xpath, $entry, $importer, $xml)
 		// Fetch the owner
 		$owner = self::fetchauthor($xpath, $entry, $importer, "dfrn:owner", true);
 
+		$owner_unknown = (isset($owner["contact-unknown"]) && $owner["contact-unknown"]);
```

Comment:
This line can be replaced by the shorter `$owner_unknown = empty($owner["contact-unknown"]);`.

Reply:
Wouldn't this invert the boolean value?

Reply:
You're right, it's 3:20 AM here and apparently I can't think straight. I updated my comment accordingly.